docs/modules/install/pages/manual.adoc

```diff
@@ -88,23 +88,31 @@ git commit -m "Initial commit. Generated with Decidim https://decidim.org"
 
 Modify your secrets (see `config/database.yml`). For this you can use https://github.com/laserlemon/figaro[figaro], https://github.com/bkeepers/dotenv[dotenv] or https://github.com/rbenv/rbenv-vars[rbenv-vars]. You should always be careful of not uploading your plain secrets on git or your version control system. You can also upload the encrypted secrets, using the sekrets gem or if you're on Ruby on Rails greater than 5.1 you can do it natively.
 
-For instance, for working with figaro, you'll need to install it:
+For a development environment, and if you are using rbenv, we strongly recommend you to use the https://github.com/rbenv/rbenv-vars[rbenv-vars] to facilitate the edition of ENV vars.
+
+First you'll need to install the rbenv-vars plugin:
 
 [source,bash]
 ----
-bundle add figaro
-bundle exec figaro install
+git clone https://github.com/rbenv/rbenv-vars.git "$(rbenv root)"/plugins/rbenv-vars
 ----
 
-Next, add this to your `config/application.yml`, using the setup PostgreSQL database name, user and, password that you've configured before.
+Then, in **any folder above your decidim generated application**, you need to create a file named `.rbenv-vars` and put your variables there:
+
+[source,bash]
+----
+nano .rbenv-vars
+----
 
-[source,yaml]
+[source,env]
```

Comment:
I'm not sure if this will be formatted correctly by asciidoc/antora, but we'll try it in a couple of minutes...
Also, maybe it'd be better to change it to the multiline string to file in bash (like `cat > EOF << .rbenv-vars` and so on), as if we're giving instruction with `nano` then we should also add the instructions to saving and quitting (Ctrl+X, Y, enter, etc) but for the moment I just want to merge this and as I have a couple other minors changes to docs in my TODO list, I'll tackle that one in another PR 😄